inference-engine/src/transformations/include/transformations/low_precision/layer_transformation.hpp

```diff
@@ -0,0 +1,380 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <algorithm>
+#include <limits>
+#include <list>
+#include <memory>
+#include <vector>
+
+#include <ngraph/ngraph.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+
+#include "iparams_manager.hpp"
+#include "ilayer_transformations_manager.hpp"
+#include "transformation_context.hpp"
+#include "quantization_details.hpp"
+#include "transformations/low_precision/common/ie_lpt_exception.hpp"
+#include "common/fake_quantize_dequantization.hpp"
+
+/*****************************************************
+ * Debug capability
+ *  - ORIGINAL_MODEL_PATH : Specify with existing folder name
+ *    to serialize original model into it (XML & BIN extensions were added)
+ *  - TRANSFORMED_MODEL_PATH : Specify with existing folder name
+ *    to serialize original model into it (XML & BIN extensions were added)
+ *  - LPT_PRINT_DEQUANTIZATION_INFO : Define it to enable
+ *    dequantization layers printing
+ *  - LPT_DISPLAY_PRECISION : Define it to to display precision info
+ *    during low precision transformations
+ *
+ *****************************************************/
+// #define LPT_ORIGINAL_MODEL_PATH "/localdisk/orig.model"
+// #define LPT_TRANSFORMED_MODEL_PATH "/localdisk/transformed.model"
+// #define LPT_PRINT_DEQUANTIZATION_INFO
+// #define LPT_DISPLAY_PRECISION
+
+namespace ngraph {
+namespace pass {
+namespace low_precision {
+
+class TRANSFORMATIONS_API DataPrecision {
+public:
+    DataPrecision() : precision(element::undefined), min(0.f), max(0.f), hasZeroPoint(false) {}
+
+    DataPrecision(const element::Type precision, const float min, const float max, const bool hasZeroPoint) :
+            precision(precision),
+            min(min),
+            max(max),
+            hasZeroPoint(hasZeroPoint) {}
+
+    static float getMinValue(const element::Type precision, const size_t levels) {
```

Comment:
This class has only static methods. Are you sure that it's best designed of this helper class?